### PR TITLE
commiting lab3 and ready to push

### PR DESCRIPTION
**Pull request by Muki-has-ideas**
pull request for lab 3. hooray

### CODE REVIEW

**Comment by Muki-has-ideas** [comment]
add version 2 will

**Comment by Muki-has-ideas** [comment]
done-zo

File: code/muki/lab03.py [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+This block was borrowed from class
+'''
+# imports just in case, I will remove these if I don't use them
+
+
+
+#   Version 1:
+numbah = input('please enter a number between 0 and 999:\n')
+
+hundreds_place = int(numbah)//100
```
Comment (brucestull) [inline code/muki/lab03.py:36]: * See [PEP8](https://peps.python.org/pep-0008/#whitespace-in-expressions-and-statements) for readability and style guidance.
* Math operators should be surrounded by single-spaces.
* Typically, a single space should follow colons in dictionaries and some commas. Check out the PEP8 documentation linked above.

File: code/muki/lab03.py [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+
+print(f'You entered:{numbah}\t Thanks for your entry.')
+print(ones_place)
+print(tens_place)
+print(hundreds_place)
+print(lose)
+
+
+single_digits = {
```
Comment (brucestull) [inline code/muki/lab03.py:50]: * For PEP8 Style Guide
* Single spaces after colons in dictionaries can help with readability.

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 3: Number to Phrase](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/03%20Number%20To%20Phrases.md)

**Needs Work**

Comments:
* User input of `510` results in improper output: Expected would be like `five hundred ten` but result was `five hundred-zero`.
* Sample output:
```
PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab03.py
please enter a number between 0 and 999:
510
You entered:510  Thanks for your entry.
0
1
5
10
510 is: five hundred-zero
PS C:\Users\Bruce\Programming\class_HB2\code\muki>
```

**Review by brucestull — CHANGES_REQUESTED** [review]
[Lab 3: Number to Phrase](https://github.com/PdxCodeGuild/class_HB2/blob/main/1%20Python/labs/03%20Number%20To%20Phrases.md)

**Needs Work**

Comments:
* Input of `108` results in improper output. See below log:
```
PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab03.py
please enter a number between 0 and 999:
108
8
You entered:108  Thanks for your entry.
8
0
1
8
PS C:\Users\Bruce\Programming\class_HB2\code\muki>
```

**Review by brucestull — CHANGES_REQUESTED** [review]
**Needs Work**

Comments:
* Entry of `999` results in two answers. Only one result should show. Please remove unnecessary print outputs. Only have one print output of final answer.
    ```
    PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab03.py
    please enter a number between 0 and 999:
    999
    99
    You entered:999  Thanks for your entry.
    9
    9
    9
    99
    999 is: ninety-nine
    999 is: nine hundred ninety-nine
    PS C:\Users\Bruce\Programming\class_HB2\code\muki>
    ```
* Entry of `101` results does not produce expected results. Expected results would be something like `one hundred one`.
    ```
    PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab03.py
    please enter a number between 0 and 999:
    101
    1
    You entered:101  Thanks for your entry.
    1
    0
    1
    1
    PS C:\Users\Bruce\Programming\class_HB2\code\muki>
    ```

**Review by brucestull — CHANGES_REQUESTED** [review]
**Needs Work**

Comments:
* Input of `101` results in `one hundred`. Expected output would be something like `one hundred one`.
```
PS C:\Users\Bruce\Programming\class_HB2\code\muki> python .\lab03.py
please enter a number between 0 and 999:
> 101
You entered:101  Thanks for your entry.
101 is: one hundred
PS C:\Users\Bruce\Programming\class_HB2\code\muki>
```

**Review by brucestull — APPROVED** [review]
**Completed**

**Pull request merged**